dozer/cogs/moderation.py

```diff
@@ -7,6 +7,7 @@
 from logging import getLogger
 
 import discord
+from discord import Forbidden
```

Comment:
why not use `discord.Forbidden` instead of a specific import?

Reply:
Because pycharm automatically imported it, nothing in particular guided that decision